firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FirebaseAppDistributionImpl.java

```diff
@@ -65,14 +66,14 @@ class FirebaseAppDistributionImpl implements FirebaseAppDistribution {
   private final ApkUpdater apkUpdater;
   private final AabUpdater aabUpdater;
   private final SignInStorage signInStorage;
-  private final SequentialReference<AppDistributionReleaseInternal> cachedNewRelease;
   private final ReleaseIdentifier releaseIdentifier;
   private final ScreenshotTaker screenshotTaker;
-  private final Executor blockingExecutor;
-  private final FirebaseAppDistributionNotificationsManager notificationsManager;
-  private TaskCache<UpdateTask> updateIfNewReleaseAvailableTaskCache = new TaskCache<>();
-  private TaskCache<Task<AppDistributionRelease>> checkForNewReleaseTaskCache = new TaskCache<>();
   @Lightweight private Executor lightweightExecutor;
+  @Blocking private final Executor blockingExecutor;
+  private final SequentialReference<AppDistributionReleaseInternal> cachedNewRelease;
+  private TaskCache<AppDistributionRelease> checkForNewReleaseTaskCache;
+  private UpdateTaskCache updateIfNewReleaseAvailableTaskCache;
+  private final FirebaseAppDistributionNotificationsManager notificationsManager;
```

Comment:
nit: keep the `final` and non-`final` (mutable) fields separate, separated by an empty line